src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -1240,6 +1240,9 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 			if (!providerId) {
 				return Promise.resolve(false);
 			}
+			if (providerId === 'sqlite') {
```

Comment:
generally it's best to avoid special-casing provides throughout the core services, as it doesn't scale well with new providers.  Is there something that can be done in the sqllite provider to better handle this command?

Reply:
That makes sense. Can we just have the changeDatabase method in sqlite provider return true instead of throwing error? https://github.com/microsoft/azuredatastudio-sqlite/blob/2cf551459fc476c44d0e5a9152de7915f94dd9ec/src/connectionProvider.ts#L57

Reply:
That would be the preferred solution. Just having it return true should be enough for the tests - if we really did want to support sqlite then we'd probably need to think more about the case here since the "database" in this case is actually the file we're connecting to but I don't think we have any logic for that. 

But just having changeDatabase be a no-op should get back the behavior we had previously and we can look into further updates later as needed.